internal/common/baseclient/baseSpeaker.go

```diff
@@ -33,7 +33,7 @@ func (s *BaseSpeaker) DecideAnnouncement(ruleId string, result bool) (string, bo
 }
 
 // CallJudgeElection is called by the legislature to decide on power-transfer
```

Comment:
Please add compulsory / optional to this. If possible, also do the same for all the functions in this file.